test/jdk/jdk/internal/vm/Continuation/Basic.java

```diff
@@ -331,8 +323,9 @@ static double noSyncFoo(int a) {
         return Integer.parseInt(r)+1;
     }
 
-    private void testPinnedNative() {
-        System.out.println("testPinnedNative");
+    // @Test
+    public void testPinnedNative() {
```

Comment:
Are you disabling this test?  if so, you can do `@Test(enabled=false)`

Reply:
I updated the test and enabled it.